src/components/Todo/todo.jsx

```diff
@@ -30,6 +34,16 @@ const Todo = () => {
 
   const isEmptyError = useSelector((state) => state.handleErrors.isEmptyError);
 
+  const currentPage = useSelector(
+    (state) => state.handleCurrentPage.currentPage
+  );
+
+  const currentTask = TASK_PER_PAGE * currentPage - isCreateBtnClicked;
```

Comment:
Let's use one uniform naming. Whether btn or button. 

```suggestion
  const currentTask = TASK_PER_PAGE * currentPage - isCreateButtonClicked;
```